src/UI/Art/SpriteArtist/PlayerSetupAiArtist.java

```diff
@@ -11,19 +11,25 @@
 import AI.AIMaker;
 import Engine.IController;
 import UI.PlayerSetupAiController;
+import UI.UIUtils;
 
 public class PlayerSetupAiArtist
 {
   private static HashMap<Integer, BufferedImage> aiNameplates = new HashMap<Integer, BufferedImage>();
   private static int maxAiNameLength = -99;
 
+  private static IController myControl;
+  private static SpriteCursor spriteCursor = new SpriteCursor(0, 0, 0, 0, UIUtils.getCOColors()[0]);
```

Comment:
Might stand to reason to make this just the default constructor with no args.

Reply:
Sensible.